### DISCUSSION TITLE
未知的异常

### DISCUSSION THREAD

**Discussion by chenhbb** [yenche123/podcast-together#23] [Q&A]
请问经常出现“开发者也遇到过这问题，目前没有解法555；我之前试过关闭浏览器重新打开，你可以试试看”，可以成功解析url，以mp3结尾，但是加载房间失败，这种一般是什么问题导致呢

**Comment by yenche123** [comment]
有链接吗？发出来看看
解析的链接和房间链接都看看

**Reply by chenhbb** [reply]
解析链接：https://media.aotuyuzhou.com/uploads/20240315/eda4c50e3829049da19c7af0ee3208d8.mp3
房间链接：https://www.podcastogether.com/room/660525562e4bd7c04c2851b8
